### ISSUE TITLE
File block: New lines between text and download button are not displayed (only Android)

### ISSUE THREAD

**Issue by fluiddot** [WordPress/gutenberg#31075]
## Description

The new lines added between the text and the download button are not displayed in the editor, however, when previewing the post, they're shown. This issue only happens on Android, on iOS the new lines are displayed properly.

## Step-by-step reproduction instructions

1. Add a file block
2. Choose any file
3. Tap on the text above the download button
4. Move the caret to the end of the text
5. Add new lines by tapping the intro button
6. Observe that the new lines are not being displayed
7. Preview the post
8. Observe that the new lines are being displayed

## Expected behaviour

New lines should be displayed when adding them at the end of the text.

## Actual behaviour

New lines are not being displayed.

## Screenshots or screen recording (optional)

https://user-images.githubusercontent.com/14905380/115690543-6fecad80-a35d-11eb-887f-f231f5e6029b.mp4

## WordPress information
- WordPress version: N/A
- Gutenberg version: N/A
- Are all plugins except Gutenberg deactivated? N/A
- Are you using a default theme (e.g. Twenty Twenty-One)? N/A

## Device information
- Device: Samsung Galaxy S20 FE 5G
- Operating system: Android 10
- WordPress app version: 17.1

**Comment by kathrynwp** [comment]
Hi @fluiddot @davidcalhoun circling back to see if this is still an issue in the latest version of WP/Gutenberg/the app, or can this issue be closed? Thanks!

**Comment by dcalhoun** [comment]
Yes, I am able to reproduce this on a Samsung Galaxy S20 (Android 12) with WP 20.5.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
